### PR TITLE
fix: Create INSTALLATION.md File

### PR DESCRIPTION
**Pull request by iamsaumya**
Fixes: #186 

**What kind of change does this PR introduce?**
- Adds INSTALLATION.md file
- Uninstall nodemon
- Tweak npm run commands

**Summary**

The PR objective is to help contributors onboard quickly by setting up the project on their localhost with proper instructions.

### CODE REVIEW

**Comment by iamsaumya** [comment]
@Sagar2366 please suggest if there is anything missing in installation.md

File: .env.sample [hunk 1]
```diff
@@ -12,7 +12,7 @@ REFRESH_TOKEN_SECRET=
 # MONGO_DB_URL=mongodb://mongo:27017/talawa-api
 # if you are using hosted mongodb, mongodb connection string will look like this
 # mongodb+srv://<username>:<password>@<server_url>/<db_name>?retryWrites=true&w=majority
-# local instance
+# for local instance use this connection string
 # mongodb://localhost:27017/<db_name>?retryWrites=true&w=majority
 
 MONGO_DB_URL=
```
[the remark this answers is not in the document]
Comment (iamsaumya) [reply .env.sample:18]: why?

[the remark this answers is not in the document]
Comment (XshubhamX) [reply .env.sample:18]: Look at line 18 what is the value of MONGO_DB_URL=,you have already specified its value above sir if im right

[the remark this answers is not in the document]
Comment (iamsaumya) [reply .env.sample:18]: L11 - L16 are comments, based on the installation approach the user can choose an appropriate connection URL.

[the remark this answers is not in the document]
Comment (iamsaumya) [reply .env.sample:18]: and will use it on L18.

[the remark this answers is not in the document]
Comment (XshubhamX) [reply .env.sample:18]: ohh ok cool Got you Sir @iamsaumya

[the remark this answers is not in the document]
Comment (iamsaumya) [reply .env.sample:18]: Glad I could help, feel free to review other changes 🍻

[the remark this answers is not in the document]
Comment (XshubhamX) [reply .env.sample:18]: @iamsaumya sir can you please have a look at my recent pull request

[the remark this answers is not in the document]
Comment (XshubhamX) [reply .env.sample:18]: ok Sir

**Comment by heyjordn** [comment]
Hi @iamsaumya, thanks for this PR, why are we removing `nodemon`?

**Comment by iamsaumya** [comment]
Hey @heyjordn , we are using pm2's watch mode in `npm run start`. it does the same thing as nodemon, so I thought it will be redundant to keep it.

**Comment by heyjordn** [comment]
@iamsaumya Ok great, everything else looks good to me. Merging

**Review by heyjordn — APPROVED** [review]
LGTM 👍

**Pull request merged**